WcaOnRails/app/models/delegate_report.rb

```diff
@@ -50,6 +50,10 @@ def posted?
     !!posted_at
   end
 
+  def can_submit_delegate_report?
+    posted? && competition.delegates.include?(current_user&.id)
```

Comment:
Does this actually work?
I'd expect you have to write `competition.delegates.map(&:id).include?(current_user&.id)` or simply `competition.delegates.include?(current_user)`.

Reply:
Yes, you were right, I missed to test in normal delegates my bad :-(
I've fixed it now